### DISCUSSION TITLE
French translation

### DISCUSSION THREAD

**Discussion by MickBsd** [Asabeneh/30-Days-Of-JavaScript#576] [Ideas]
Hello, i would like translate this awesome  project in french.

A team has already begin this ?

**Comment by Asabeneh** [comment]
I have accepted your request. Good luck!

**Comment by MickBsd** [comment]
Really cool :) 

I start that tonight !

**Reply by Asabeneh** [reply]
You may try to finish it before 2023 if you have time.
